lib/heimdall_tools/cli.rb

```diff
@@ -99,6 +99,18 @@ def jfrog_xray_mapper
       puts "#{options[:output]}"
     end
 
+    desc 'aws_config_mapper', 'aws_config_mapper pulls Ruby AWS SDK data to translate AWS Config Rule results into HDF format Json to be viewable in Heimdall'
+    long_desc Help.text(:aws_config_mapper)
+    option :custom_mapping, required: false, aliases: '-m'
+    option :output, required: true, aliases: '-o'
+    option :verbose, type: :boolean, aliases: '-V'
+    def aws_config_mapper
+      hdf = HeimdallTools::AwsConfigMapper.new(options[:addl_mapping]).to_hdf
```

Comment:
@jkufro I think you intended it to be  `options[: custom_mapping]` and not `options[:addl_mapping]`

Reply:
done